src/lib/util.ts

```diff
@@ -6,7 +6,7 @@ import {
   IProposalState,
   IRewardState,
   ISchemeState } from "@daostack/client";
-import { GenericSchemeRegistry } from "genericSchemeRegistry";
+import { GenericSchemeRegistry } from "../genericSchemeRegistry";
```

Comment:
isn't this fixed by the change you made in package.json?

Reply:
ah yeah that's from before the fix